packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -155,7 +155,12 @@ export function useInnerBlocksProps( props = {}, options = {} ) {
 		[ clientId, isSmallScreen ]
 	);
 
-	const ref = useMergeRefs( [ props.ref, fallbackRef ] );
+	const ref = useMergeRefs( [
+		props.ref,
+		useBlockDropZone( {
```

Comment:
Just to clarify my own understanding, you basically avoided the "wrapperRef" prop by calling this in both inner blocks and the root list right?

Reply:
That's correct. And in the future, we should be able to reuse `useInnerBlocksProps` in the `BlockList` component to avoid some code duplication.